install-cli.sh

```diff
@@ -21,6 +21,7 @@ set -o errexit
 set -o pipefail
 
 REPO="livekit-cli"
+BIN_NAME="lk"
```

Comment:
let's update the script last. the main reason is that whatever is merged in main is going to be pulling the last release from GH. so if we did it now it'll break previous installs

Reply:
Good idea -- I'll split off the install script changes into a separate PR. Can land that and docs site changes around the same time.

Like the idea of a dummy `livekit-cli` binary too!